common/source/docs/common-flywoo-f745.rst

```diff
@@ -42,17 +52,28 @@ Specifications
    -  Built-in OSD
 
 
--  **Size and Dimensions**
+   **Size and Dimensions AIO**
 
-   - 33.5mm x 333.5mm (25.6 x 25.6mm mount pattern)
+   - 33.5mm x 33.5mm (25.6 x 25.6mm mount pattern)
    - 8.5g
 
-Where to Buy
-============
+   **Size and Dimensions Nano**
+   
+   - 22mm x 235mm (26mm x16mm mount pattern)
```

Comment:
This is 16mmx16mm mount pattern I believe

Reply:
typo...thanks